packages/www/components/Dashboard/WebhookDialog/index.tsx

```diff
@@ -220,7 +220,8 @@ const WebhookDialog = ({
               css={{
                 width: "100%",
                 borderRadius: 6,
-                height: 200,
+                height: "auto",
```

Comment:
let's change height to 200 and add overflow: 'hidden' so the container doesn't move as you add items.

Reply:
changed it